cloudinit/util.py

```diff
@@ -1296,6 +1310,8 @@ def find_devs_with(criteria=None, oformat='device',
     elif is_OpenBSD():
         return find_devs_with_openbsd(criteria, oformat,
                                       tag, no_cache, path)
+    elif is_DragonFlyBSD():
+        return ['/dev/cd0']
```

Comment:
i don't think we an remove the WIP label here, before we haven't found out how this actually works

Reply:
I would like to land https://github.com/canonical/cloud-init/pull/298 first.